src/main/java/net/snowflake/ingest/streaming/internal/ParquetRowBuffer.java

```diff
@@ -185,14 +190,23 @@ public void setupSchema(List<ColumnMetadata> columns) {
 
         this.statsMap.put(
             columnPath,
-            new RowBufferStats(columnPath, null /* collationDefinitionString */, ordinal, fieldId));
+            new RowBufferStats(
+                columnPath,
+                null /* collationDefinitionString */,
+                ordinal,
+                fieldId,
+                clientBufferParameters.getIsIcebergMode()));
```

Comment:
lets add a clientBufferParameters.getEnableDistinctValues() instead of getIsIcebergMode.

Reply:
Added.